packages/worker/src/runtime.ts

```diff
@@ -63,7 +63,7 @@ export interface CompiledRuntimeOptions {
 
 function defaultTelemetryOptions(): TelemetryOptions {
   return {
-    tracingFilter: 'temporal_sdk_core=WARN',
+    tracingFilter: 'temporal_sdk_core=INFO',
```

Comment:
Is this change by intention?

Reply:
Yes, it is, meant to call attention to that. The context variables are emitted only at info level. So, default log level needs to be up to there or even warning events won't have context in the console output. This is... semi unfortunate but just the way it is. There's very few things emitted at info level so I think it's a reasonable change to default.